src/hooks/useConfirmationText.ts

```diff
@@ -0,0 +1,22 @@
+import { useEffect, useState } from 'react';
+import { useHistory, useLocation } from 'react-router-dom';
+
+function useConfirmationText() {
```

Comment:
It would be nice to add some documentation that describes what this hook does, with bonus points for mentioning how a developer would setup `location.state` so that it can access that value.